### PR TITLE
add initial integ test workflows

### PR DESCRIPTION
**Pull request by wesk**
*Description of changes:*

Wire up the quick tests to run on GitHub Actions with each PR. 

Observed that the accuracy values differ between Mac and Linux 

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: test/integration_tests/run_quick_test.py [hunk 1]
```diff
@@ -65,4 +65,9 @@
     else:
         accuracies = []
 
+    print("Expected Accuracy:", test_config["expected_accuracy"])
+    print("Actual Accuracy:", accuracies)
```
Comment (wistuba) [inline test/integration_tests/run_quick_test.py:69]: only print in case of failure

File: .gitignore [hunk 2]
```diff
@@ -0,0 +1,8 @@
+**/__pycache__/
```
Comment (wistuba) [inline .gitignore:1]: can we drop this file?

**Review by wistuba — COMMENTED** [review]
I have only minor comments

File: .gitignore (hunk 2 above)
Comment (wesk) [reply .gitignore]: Sure, I pulled it out into a separate PR: https://github.com/awslabs/Renate/pull/190

**Review by wistuba — APPROVED** [review]

**Pull request merged**